examples/jsm/webxr/Text2D.js

```diff
@@ -0,0 +1,38 @@
+import * as THREE from "../../../build/three.module.js";
+
+function createText(message, height) {
+  var canvas = document.createElement("canvas");
```

Comment:
Do you mind using `let`/`const` in this file?

Reply:
Fixed!